.teamcity/_self/projects/WPComTests.kt

```diff
@@ -330,6 +331,172 @@ fun jetpackBuildType(screenSize: String): BuildType {
 	}
 }
 
+private object GutenbergPlaywrightE2E: BuildType {
```

Comment:
I think this is the correct syntax, but not sure: 

```suggestion
private object GutenbergPlaywrightE2E: BuildType({
```

Reply:
> > Would this assumption be true
> 
> I'm honestly not sure. I think our approach so far is to merge a small PR creating a very minimal build, and then do all of the main work after that exists

Yeah, good call.